packages/sdk/src/wallet/Ordit.ts

```diff
@@ -98,17 +98,12 @@ export class Ordit {
     this.#network = value
   }
 
-  getAddressByType(type: AddressFormats) {
+  getAddressByType(type: AddressFormats, accountIndex: number, addressIndex: number) {
```

Comment:
Maybe we should not change this function? let it still return an array of address by type. But create a new func like `getAddress` that takes in type, accountIndex and addressIndex which returns exactly 1 address data if matched.